database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Models/PostViewModel.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  Copyright (c) 2021 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import Firebase
+
+class PostViewModel: ObservableObject, Identifiable {
+  @Published var id: String
```

Comment:
nit: View Model should expose only properties required for View. Is `id` displayed anywhere? The same question is relevant for the rest of the published properties.